vernac/comSearch.ml

```diff
@@ -105,11 +105,11 @@ let () =
       optread  = (fun () -> !search_output_name_only);
       optwrite = (:=) search_output_name_only }
 
-let interp_search env sigma s r =
+let interp_search ?(print_loc=false) env sigma s r =
```

Comment:
It would be nice to turn this on and commit the output test changes so we can see the effect.

Reply:
Actually it should be turned on, I broke the checker tho so this is why it doesn't show.

Reply:
Here you go, silly typo in values.ml